examples/utils/get-tinylicious-container/src/getTinyliciousContainer.ts

```diff
@@ -25,7 +25,7 @@ import { getContainer } from "./getContainer";
  * URL shape, so it expects the request url to have this format (as opposed to a more traditional URL):
  * documentId/containerRelativePathing
  */
-class InsecureTinyliciousUrlResolver implements IUrlResolver {
+export class InsecureTinyliciousUrlResolver implements IUrlResolver {
```

Comment:
TBH I'm a little worried about the proliferation of these utilities.  I created `getTinyliciousContainer` as a bandaid originally because working with the url resolver is painful and not necessary for most scenarios, and so I intentionally did not expose it.  So if working with the url resolver is necessary for a scenario, then perhaps it would be better to step away from these utils and do something more legit.  I think this might slowly devolve back into the original `InsecureUrlResolver` otherwise.

I wouldn't object to the change necessarily, but I think it's more evidence that we need a better permanent solution sooner than later.

Reply:
I was having some trouble with the plain `InsecureUrlResolver` with tinylicious information that was there before, but that appears to be because I just constructed the request wrongly, so it can keep using that.  It's still _very_ similar to what's happening here, though I'm not sure that contributes further to needing the better permanent solution then?

Reply:
Ended up being able to use `getTinyliciousContainer` for one spot, and kept the old `getTinyliciousResolver` for the other (though the existing `getTinyliciousResolver` is basically a dupe of `InsecureTinyliciousUrlResolver`)